panoramapublic/resources/schemas/dbscripts/postgresql/panoramapublic-22.000-22.001.sql

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright (c) 2019 LabKey Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+CREATE TABLE panoramapublic.DataValidation
+(
+    _ts TIMESTAMP,
+    CreatedBy              USERID,
+    Created                TIMESTAMP,
+    ModifiedBy             USERID,
+    Modified               TIMESTAMP,
+
+    Id                         SERIAL NOT NULL,
+    ExperimentAnnotationsId    INT    NOT NULL,
+    JobId                      INTEGER,
+    Status                     INT,
+
+    CONSTRAINT PK_DataValidation PRIMARY KEY (Id),
+    CONSTRAINT FK_DataValidation_ExperimentAnnotations FOREIGN KEY (ExperimentAnnotationsId) REFERENCES panoramapublic.ExperimentAnnotations(Id)
+);
+CREATE INDEX IX_DataValidation_ExperimentAnnotations ON panoramapublic.DataValidation(ExperimentAnnotationsId);
+CREATE INDEX IX_DataValidation_JobId ON panoramapublic.DataValidation(JobId);
+
+
+CREATE TABLE panoramapublic.SkylineDocValidation
+(
+    Id                         SERIAL NOT NULL,
+    ValidationId               INT NOT NULL,
+    RunId                      BIGINT NOT NULL, -- targetedms.runs.Id
+    Container                  ENTITYID NOT NULL, -- Container where the run lives. This could be a subfolder of the main experiment folder
+    Name                       VARCHAR(300) NOT NULL, -- Name of the Skyline file
+
+    CONSTRAINT PK_SkylineDocValidation PRIMARY KEY (Id),
+    CONSTRAINT FK_SkylineDocValidation_DataValidation FOREIGN KEY (ValidationId) REFERENCES panoramapublic.DataValidation(Id)
```

Comment:
Should probably have a FK on container too

Reply:
I removed the container column.  Getting the container from the targetedms run instead.